sdks/python/apache_beam/pipeline.py

```diff
@@ -1071,6 +1071,14 @@ def named_inputs(self):
     }
     return dict(main_inputs, **side_inputs)
 
+  def main_inputs(self):
```

Comment:
Generic transforms don't have the notion of main inputs, let's filter things out in the implementation in ParDo.